services/centralized-grafana/33.1.3/centralized-grafana.yaml

```diff
@@ -38,6 +38,12 @@ rules:
     verbs:
       - get
       - head
+  - nonResourceURLs:
```

Comment:
🤔 can you double check why we define these roles separately here vs using the ones created in the chart? (It was very likely me who did this, I can’t recall if there was a reason why or if it was done mistakenly)

Reply:
good catch! I took a look at the values of centralized and found: https://github.com/mesosphere/kommander-applications/blob/main/services/centralized-grafana/33.1.1/defaults/cm.yaml#L13
Let me try flipping that to true..